### ISSUE TITLE
Notebooks does not separate examples

### ISSUE THREAD

**Issue by MichielStock** [MichielStock/Julia925#25]
Many blocks of code are not represented well in the notebooks, e.g.

```julia
1 + 2       # adding integers

1.0 + 2.0   # adding floats

2 / 4       # standard division

div(2, 4)   # Computes 2/4 truncated to an integer

35 \ 7      # inverse division

1 // 3      # fractions

1//2 + 1//4

'c'        # characters (unicode)

:symbol    # symbols, mostly used for macros
```

is in a single cell, hence it only prints the last command. This is not didactic. Either every line should be split in a separate code block in the .jmd file or there is some Weave command to do this?

Should be solved for
- [ ] basics
- [ ] collections
- [x] types
- [x] CA
- [x] error prop

**Comment by Beramos** [comment]
Ik doe het op maandag je mag mij assignen heb mijn laptop niet mee dit
weekend

Op za 14 dec. 2019 om 12:53 schreef Michiel Stock <notifications@github.com>

**Comment by MichielStock** [comment]
Related, in collections @Beramos uses `@show` a lot (to circumvent this problem). I don't find this good practice. Remove this and split the blocks (if no weave command is found).

**Comment by Beramos** [comment]
Parking this idea for now

**Comment by Beramos** [comment]
@Michiel this was fixed with `term=true` in #26, right?

**Comment by MichielStock** [comment]
Nope, you have to split the code blocks manually.

term=true solves it for pdf, not notebooks
